drivers/sensor/nuvoton_adc_cmp_npcx/Kconfig

```diff
@@ -0,0 +1,15 @@
+#  ADC CMP NPCX driver configuration options
+
+# Copyright (c) 2022 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+if ADC_NPCX
+
+config ADC_CMP_NPCX
+	bool "Nuvoton NPCX ADC threshold detection interruption"
+	default n
```

Comment:
`default n` is the default value for booleans. there's no reason to write it out

Reply:
Done.